### PR TITLE
chore: Use bitnami docker image for kubectl

### PR DESCRIPTION
**Pull request by souleb**
This will enable having dependabot automatic updates for docker images.

### CODE REVIEW

**Review by almaslennikov — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -51,15 +47,19 @@ RUN --mount=type=cache,target=/go/pkg/mod \
...
 
 ARG ARCH
 
 WORKDIR /
 COPY --from=builder /workspace/manager .
-COPY --from=builder /workspace/kubectl-${ARCH} /usr/local/bin/kubectl
 COPY --from=builder /workspace/crds /crds
 
+# Copy the kubectl binary
+COPY --from=kubectl /opt/bitnami/kubectl/bin/kubectl /usr/local/bin/
```
Comment (e0ne) [inline Dockerfile:61]: We use UBI base image. bitnami/kubectl is debian-based. I don't think it's a good option to mix OS'ws

**Review by e0ne — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (souleb) [reply Dockerfile:61]: Can you tell more about that?

Here we just copy the Kubectl binary. Kubectl is written in Golang, so it's self-contained. Did I miss anything?

Comment (e0ne) [reply Dockerfile:61]: if we use only one binary it's ok

**Review by e0ne — APPROVED** [review]

**Review by adrianchiris — APPROVED** [review]
LGTM.

@e0ne  BTW didnt we want to switch to a client-go binary to avoid using kubectl ?

**Comment by e0ne** [comment]
We don't need this once #1118 will be merged

**Pull request closed without merging**